lib/repack/tests/test_repack.py

```diff
@@ -179,7 +179,7 @@ def test_repack_float_object_to_float32():
 
 def test_repack_category():
     s = pd.Series(["a", "b", "c", None])
-    assert s.dtype == np.object_
+    assert s.dtype == np.object_ or s.dtype == "str"
```

Comment:
With pandas 3.0, the above series creation will infer the dtype to be string, and no longer object dtype